styles/globals.css

```diff
@@ -18,4 +18,14 @@ body {
     color: var(--text-color);
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
-}
\ No newline at end of file
+}
+
+.title-link {
+    font-size: 1.5rem;
+    text-decoration: underline;
+    color: inherit;
+}
+
+.title-link:hover {
+    color: #007bff;
+}
```

Comment:
Check if it's possible for us to use Tailwind here instead of writing CSS. Helps to keep things simple.